intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -214,6 +216,70 @@ def convert_date(value):
     return harmonization.DateTime.sanitize(value)
 
 
+# https://www.shadowserver.org/wiki/pmwiki.php/Services/Open-DB2
+scan_db2 = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip'),
+        ('source.port', 'port')
+    ],
+    'optional_fields': [
+        ('protocol.transport', 'protocol'),
+        ('source.reverse_dns', 'hostname'),
+        ('source.asn', 'asn'),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('destination.reverse_dns', 'db2_hostname'),
```

Comment:
db2_hostname is the local hostname, not a fqdn nor a destination. So we should probably put this into extra?

Reply:
Yes, you're right. I will include to extra field.